src/main/java/seedu/duke/parsers/AddParser.java

```diff
@@ -19,34 +19,38 @@
     private static final String TASK_MODULE = StringConstants.TASK_MODULE;
     private static final String MODULE_CODE = StringConstants.MODULE_CODE;
     private static final String MODULE_DESCRIPTION = StringConstants.MODULE_DESCRIPTION;
+    private static final String MODULAR_CREDIT = StringConstants.MODULAR_CREDIT;
 
     // Unescaped regex for testing (split across a few lines):
-    // (/t\s+\"(?<taskName>[^\"]+)\"(\s+-d\s+\"(?<taskDescription>[^\"]+)\")?
-    // (\s+-t\s+\"(?<estimatedWorkingTime>[^\"]+)\")?(\s+-m\s+\"(?<taskModule>\w+)\")?
-    // |/m\s+(?<moduleCode>\w+?(?=(\s+-d\s+)|$))(\s+(-d\s+\"(?<moduleDescription>.+)\"))?)
+    // (/t\s+\"(?<taskName>[^\"]+)\"(\s+-d\s+\"(?<taskDescription>[^\"]+)\")?(\s+-t\s+\"
+    // (?<estimatedWorkingTime>[^\"]+)\")?(\s+-m\s+\"(?<taskModule>\w+)\")?|/m\s+(?<moduleCode>\w+?)(\s+(/c\s+
```

Comment:
The other line (line 35) was corrected to remove the "" from the commented out regex, but not here. Let's fix it so we don't get confused during regex testing later.

(Admittedly this problem was my fault so good that you caught that)

Reply:
Yup, I have corrected both regex in the code and the explanation.